airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -206,9 +206,7 @@ def _clean_date(value: datetime | None) -> str:
 
         https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html#_reserved_characters
         """
-        if value is None:
-            return ""
-        return value.strftime("%Y_%m_%dT%H_%M_%S_%f")
+        return value.strftime("%Y_%m_%dT%H_%M_%S_%f") if value else ""
```

Comment:
Doesn’t seem particularly necessary to me, we’re not playing code golf…

Reply:
Same for other fast return checks below.

Reply:
Reverted all such cases.